internal/test_dataset_decompression.py

```diff
@@ -104,6 +104,7 @@ def _restrict_metadata_to_downloaded_files(big_metadata_file, download_folder):
 def _test_iterate_dataset(
     download_folder,
     download_modalities=None,
+    download_small_subset=False,
```

Comment:
Above, small subset refers to the 52-video example, but here it is used as 3-category debug. Align one way or another?